### ISSUE TITLE
Out of Memory and weird display 

### ISSUE THREAD

**Issue by selsie** [TES5Edit/TES5Edit#1241]
Using 4.0.4 without any problem I am just checking out the new, fresh, improved (?) version 4.1.5

The listview to the left looks squished (4k display) - To fix 4k display problems, one can easily adjust the compatibility settings in properties of the .exe in "High DPI Settings" and select "System Advanced". Please add that info on the Nexus page.

Now for the repeatable Out of Memory error, that didn't ever show up in 4.0.4

````
*** Application started *** 
Time: [522103468] 2023-11-04 10:52:02.354
ExceptionInfo: nil
Exception: EOSError
Message: System Error.  Code: 8.
Zur Verarbeitung dieses Befehls sind nicht genügend Speicherressourcen verfügbar
Filesize: 24253952
Filedate: 2023-11-04 07:55:36
Computer: x
User: x
ThreadID: 28628
ProcessID: 19240
Version: SSEEdit 4.1.5 EXPERIMENTAL
--------------------------------------------------------------
Callstack (Frames):
(00037033) [003D8033]
[003D8033] System.SysUtils.RaiseLastOSError (Line 27338, "System.SysUtils.pas")
[00BB7788] wbImplementation.TwbFile.Create
[00C0608C] wbImplementation.wbFile
[0145A53F] xeMainForm.TLoaderThread.Execute
[0047B93D] System.Classes.ThreadProc (Line 15845, "System.Classes.pas")
[003AB338] System.ThreadWrapper (Line 25778, "System.pas")
==============================================================
```
`
![sseedit 4 1 5](https://github.com/TES5Edit/TES5Edit/assets/15048308/8bbd2637-2206-4bf0-9c82-c39dd5124785)

The out of Memory appears always at the exact same mod (SOTFull.esp).

Please let me know which Information you need to fix this.

**Comment by ElminsterAU** [comment]
Your report does not provide any information that could be used to reproduce the problem.

**Comment by selsie** [comment]
> Your report does not provide any information that could be used to reproduce the problem.

Check out the attached screenshot.

**Comment by ElminsterAU** [comment]
There is no attached screenshot. 

Any issues with the text size you might have should be a result of incorrect information stored in your settings file, possibly from a previous version. You can reset that the settings by holding Ctrl+Alt+Shift before the main form shows and you will get a dialog asking you if you want to reset the settings.

For the out of memory error, it definitely doesn't occur when just loading the vanilla (base game + all offical DLC) files. So, please provide me the actual files I need to load to reproduce this problem.

**Comment by ElminsterAU** [comment]
Your load order might also simply be too large for a 32bit process, in which case using the 64bit version should be able to load everything.

**Comment by selsie** [comment]
> There is no attached screenshot.
> 
> Any issues with the text size you might have should be a result of incorrect information stored in your settings file, possibly from a previous version. You can reset that the settings by holding Ctrl+Alt+Shift before the main form shows and you will get a dialog asking you if you want to reset the settings.
> 
> For the out of memory error, it definitely doesn't occur when just loading the vanilla (base game + all offical DLC) files. So, please provide me the actual files I need to load to reproduce this problem.


https://github.com/TES5Edit/TES5Edit/assets/15048308/8bbd2637-2206-4bf0-9c82-c39dd5124785

Here is the screenshot.

The loadorder works with 4.0.4 32bit, and there is no 64 bit exe of 4.1.5

**Comment by ElminsterAU** [comment]
I don't care if it loads in 32bit in 4.0.4.

4.1.5 is a completely different version, with years of development time and additions, which might use more memory per loaded record, or there might be a difference in memory layout or 10000 other possible reasons.

A 64bit version is included in both the downloads on NexusMods and on GitHub.

**Comment by ElminsterAU** [comment]
Also, it's very obvious from the screenshot that the problem happens when loading DepthsOfSkyrim.esp, not SOTFull.esp

**Comment by selsie** [comment]
Thanks for your help.

I found the 64 bit version in the optional folder. It loaded my modlist just fine (I got an information overload, you changed quite a bit there), but it took much longer than I'm used to with the old version.

The squished text in the listviews can indeed be fixed in the UI settings, but the UI settings don't change the size and shape of script generated forms, so there's no way to avoid adjusting the high DPI settings.

https://github.com/TES5Edit/TES5Edit/assets/15048308/546287f0-76ff-462d-a4dd-aa3179accae5
This screenshot shows a window suffering from missing DPI adjustment.

Everything seems to work now.
